src/ORM/RulesChecker.php

```diff
@@ -92,14 +92,15 @@ public function isUnique(array $fields, array|string|null $message = null): Rule
      *
      * @param array<string>|string $field The field or list of fields to check for existence by
      * primary key lookup in the other table.
-     * @param \Cake\ORM\Table|\Cake\ORM\Association|string $table The table name where the fields existence will be checked.
+     * @param \Cake\ORM\Table|\Cake\ORM\Association|string $associated The alias name of the table
+     *   where the fields existence will be checked.
      * @param array<string, mixed>|string|null $message The error message to show in case the rule does not pass. Can
      *   also be an array of options. When an array, the 'message' key can be used to provide a message.
      * @return \Cake\Datasource\RuleInvoker
      */
     public function existsIn(
         array|string $field,
-        Table|Association|string $table,
+        Table|Association|string $associated,
```

Comment:
Do we need to rename the parameter? This is kind of a breaking change with named parameters.

Reply:
The name `table` was tricking me into thinking I have to pass a Table name. Twice now. So in my opinion the name is wrong, since it does not expect a table name but the name of the association.

But I didn't think of a named parameter tbh. It's up to you guys how to proceed with this. My PR is just a suggestion.